src/Packetery/Module/Framework/WcTaxTrait.php

```diff
@@ -9,30 +9,27 @@
 
 namespace Packetery\Module\Framework;
 
+use WC_Tax;
+
 /**
  * Trait WcTaxTrait.
  *
  * @package Packetery
  */
 trait WcTaxTrait {
-	/**
-	 * Gets an array of matching shipping tax rates for a given class.
-	 *
-	 * @return array
-	 */
+	public function taxGetRates( string $taxClass, ?object $customer = null ): array {
+		return WC_Tax::get_rates( $taxClass, $customer );
+	}
+
 	public function taxGetShippingTaxRates(): array {
-		return (array) \WC_Tax::get_shipping_tax_rates();
+		return (array) WC_Tax::get_shipping_tax_rates();
```

Comment:
Pole čeho? Brambor? :D  nešlo by ty pole striktně otypovat v tomhle traitu?

Reply:
Vypadá to, že spolehlivě vrací array, takže je to ochrana proti zavádějícímu phpdocu.

Reply:
spíš jsem to myslel tak, že bych tam rád viděl třeba array<array,  null|string> nebo co tam může být